### PR TITLE
Issue 18

### PR DESCRIPTION
**Pull request by bs610**
Add a Fake_ESP32Serial class that can be used for testing when an arduino isn't present. Used if read_from_esp is False in the YAML.

### CODE REVIEW

File: gui/communication/fake_esp32serial.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        Get command wrapper
+
+        arguments:
+        - name           the parameter name as a string
+
+        returns: the requested value
+        """
+
+        with self.lock:
+            if name in self.set_params:
```
Comment (carmelopellegrino) [inline gui/communication/fake_esp32serial.py]: Could you please make the function return a string?

File: gui/mvm_gui.py [hunk 2]
```diff
@@ -4,26 +4,34 @@
...
+import os
+import os.path
 from PyQt5 import QtCore, QtGui, QtWidgets
 from PyQt5 import uic
 
 import yaml
 
 from mainwindow import MainWindow
 from communication.esp32serial import ESP32Serial
+import communication.fake_esp32serial
```
Comment (carmelopellegrino) [inline gui/mvm_gui.py]: could you please use the same import style of the non-fake ESP32Serial?

Comment (bs610) [reply gui/mvm_gui.py]: Addressed in d0645a0.

File: gui/communication/fake_esp32serial.py (hunk 1 above)
Comment (bs610) [reply gui/communication/fake_esp32serial.py]: Addressed in d0645a0.

**Comment by carmelopellegrino** [comment]
Perfect, tested against the new master (merged with issue-18). Thank you!

**Pull request merged**